core/build.gradle

```diff
@@ -21,6 +21,7 @@ managedDependencies {
         compile "io.netty:$it"
     }
     compile 'io.netty:netty-transport-native-epoll:linux-x86_64'
+    compile 'io.netty:netty-transport-native-kqueue:osx-x86_64'
```

Comment:
It's not a huge deal, but I think it depends on the use case - if it's not used 99.9% of the time, it seems wasteful. I've never heard of anyone running servers on Mac in production at scale (I assume they exist but seems really rare). With BoringSSL, it's an easier decision since it actually enables a feature, HTTP2 without alpn weirdness which is a significant improvement to development, but a kqueue perf boost during development doesn't seem like a big deal IMO.